src/app/report/edit-display/edit-display.component.ts

```diff
@@ -138,43 +149,28 @@ export class EditDisplayComponent {
 
   getDifferences(): ReportDifference[] {
     return [
-      this.getDifference('name', this.report.name, this.editFormComponent.name),
-      this.getDifference('description', this.report.description, this.editFormComponent.description),
-      this.getDifference('path', this.report.path, this.editFormComponent.path),
-      this.getDifference('transformation', this.report.transformation, this.editFormComponent.transformation),
-      this.getDifference('variables', this.report.variables, this.editFormComponent.variables),
+      this.editFormComponent.getDifference('name'),
+      this.editFormComponent.getDifference('description'),
+      this.editFormComponent.getDifference('path'),
+      this.editFormComponent.getDifference('transformation'),
+      this.editFormComponent.getDifference('variableCsv'),
     ];
   }
```

Comment:
Might be better to move all of this to the editFormComponent, that way the edit-display component doesn't need to know these string values